onboarding_auth/onboardingAuthData/src/main/java/com/alexafit/onboardingauthdata/remote/TaskyApi.kt

```diff
@@ -0,0 +1,31 @@
+package com.alexafit.onboardingauthdata.remote
+
+import com.alexafit.onboardingauthdata.BuildConfig
+import com.alexafit.onboardingauthdata.model.remote.Login
+import com.alexafit.onboardingauthdata.model.remote.LoginResponse
+import com.alexafit.onboardingauthdata.model.remote.Register
+import com.alexafit.onboardingauthdata.model.remote.RegisterResponse
+import retrofit2.Response
+import retrofit2.http.Body
+import retrofit2.http.Header
+import retrofit2.http.POST
+
+interface TaskyApi {
+    @POST("register")
+    suspend fun registerUser(
+        @Header("x-api-key") apiToken: String = BuildConfig.TASKY_API_KEY,
```

Comment:
This apikey will be added to every single request
So would be better to have an interceptor, so it adds it automatically